src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2268,6 +2268,8 @@ type Trip implements Node {
   "Whether bikes are allowed on board the vehicle running this trip"
   bikesAllowed: BikesAllowed
   blockId: String
+  "Whether cars are allowed on board the vehicle running this trip"
+  carsAllowed: CarsAllowed
```

Comment:
Do you actually need this information in your UI? If not I would remove that field because we don't know how the standardisation process will play out.

Reply:
I removed this